packages/web/components/PageHeader.tsx

```diff
@@ -1,28 +1,80 @@
 import { Box, Button, Flex, Image, Stack } from '@metafam/ds';
+import MetaBoxButton from 'assets/drawer/box.button.bg.png';
+import MetaBox from 'assets/drawer/desktop.box.png';
+import MetaDrawer from 'assets/drawer/desktop.gradient.png';
+import MetaForum from 'assets/drawer/forum.png';
+import MetaPlayers from 'assets/drawer/players.png';
+import MetaQuests from 'assets/drawer/quests.png';
+import MetaRaids from 'assets/drawer/raids.png';
+import MetaGameLogo from 'assets/logo.png';
 import MetaGameImage from 'assets/metagame.png';
 import { MetaLink } from 'components/Link';
 import { LoginButton } from 'components/LoginButton';
 import React from 'react';
 
+import { DrawerItems } from './DrawerItems';
+
 const MenuItem: React.FC<React.ComponentProps<typeof MetaLink>> = ({
   children,
   href,
   isExternal,
 }) => {
   return (
     <MetaLink
+      zIndex="2"
+      href={href}
+      isExternal={isExternal}
+      textDecoration="none"
+      _hover={{ textDecoration: 'none' }}
+    >
+      <Button
+        display="flex"
+        flexDirection="column"
+        justifyContent="center"
+        alignItems="center"
+        width="calc(32rem / 5)"
```

Comment:
Should use responsive styles and not hard coded ["magic numbers"](https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544)

Reply:
Thanks for the link! I need to review this.